### PR TITLE
Fix frontend urls

### PR DESCRIPTION
**Pull request by tbwrigh**
We had many localhosts in our app, they be gone now I think

### CODE REVIEW

File: frontend-admin/src/pages/vans/vans-page.tsx [hunk 1]
```diff
@@ -28,7 +30,7 @@ const VanPage: React.FC = () => {
   const createVan = async (vanData: VanData) => {
     // Convert the routeId to an integer
     try {
-      await fetch('http://localhost:8000/vans/', {
+      await fetch(baseUrl + '/vans/', {
```
Comment (OxygenCobalt) [inline frontend-admin/src/pages/vans/vans-page.tsx:33]: Nit: Inconsistent use of format strings and string concat

**Review by OxygenCobalt — CHANGES_REQUESTED** [review]
Otherwise looks good

File: .env [hunk 2]
```diff
@@ -0,0 +1,10 @@
+POSTGRES_PASSWORD=password
+POSTGRES_USER=postgres
+POSTGRES_DB=postgres
+DATABASE_URL=postgresql://postgres:password@database:5432/postgres
+CACHE_TYPE=ttl
+CACHE_TTL=120
+CACHE_MAXSIZE=1000
+VITE_BACKEND_URL=http://localhost:8000
+REACT_APP_ADMIN_URL=http://localhost:3000
+REACT_APP_ADMIN=true
```
Comment (OxygenCobalt) [inline .env:10]: So we're going to have just one big .env file? Is that good practice?

**Review by OxygenCobalt — CHANGES_REQUESTED** [review]

File: .env (hunk 2 above)
Comment (tbwrigh) [reply .env]: ehhh with our size it probably doesn't matter but we'll split it

File: frontend-admin/src/pages/vans/vans-page.tsx (hunk 1 above)
Comment (tbwrigh) [reply frontend-admin/src/pages/vans/vans-page.tsx]: sure, we can fix

**Pull request merged**